webpack.config.js

```diff
@@ -17,20 +19,50 @@ module.exports = (env) => {
           use: ["babel-loader"],
         },
         {
-          test: /\.(css|sass)$/i,
-          use: ["style-loader", "css-loader", "sass-loader"],
+          test: /\.css$/i,
+          use: [MiniCssExtractPlugin.loader, "css-loader"],
         },
         {
-          test: /\.(png|svg|jpg|jpeg|gif)$/i,
+          test: /\.sass$/i,
+          use: ["sass-loader"],
+        },
+        {
+          test: /\.(png|jpg|jpeg|gif)$/i,
           type: "asset/resource",
         },
+        {
+          test: /\.svg$/i,
+          use: ["@svgr/webpack"],
+        },
+        {
+          test: /\.js$/,
+          enforce: "pre",
+          use: ["source-map-loader"],
```

Comment:
Well, webpack has its own source-maps, and I would suggest to use those instead of setting up some custom loader - https://webpack.js.org/configuration/devtool/